README.md

```diff
@@ -68,7 +68,7 @@ import * as PIXI from 'pixi.js'
 #### CDN Install (via cdnjs)
 
 ```html
-<script src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/6.5.7/pixi.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/6.5.7/browser/pixi.min.js"></script>
```

Comment:
This link isn't working.
Alternatively: https://unpkg.com/pixi.js/dist/browser/pixi.min.js

Reply:
6.5.7 is not out yet :) [https://cdnjs.cloudflare.com/ajax/libs/pixi.js/6.5.6/browser/pixi.min.js](https://cdnjs.cloudflare.com/ajax/libs/pixi.js/6.5.6/browser/pixi.min.js) is available!

Reply:
Perhaps we can provide multiple CDN choices here?

Reply:
I like unpkg.com and jsdelivr.net because you don't need to provide the version to get the latest tag. At least for our purposes, it makes these readme's a little more maintainable.

https://cdn.jsdelivr.net/npm/pixi.js/dist/browser/pixi.min.js
https://unpkg.com/pixi.js/dist/browser/pixi.min.js

Reply:
Nice!